src/components/inputs/time-input/time-input-body.js

```diff
@@ -20,7 +20,7 @@ const getIconTheme = (isDisabled, isMouseOver) => {
 export const ClearSection = props => (
   <div
     onClick={props.isDisabled || props.isReadOnly ? undefined : props.onClear}
-    css={getClearSectionStyles(props)}
+    css={theme => getClearSectionStyles(props, theme)}
```

Comment:
If only if was curried :)

Reply:
at least I have VRT to catch my mistakes 🙈